app/src/main/AndroidManifest.xml

```diff
@@ -54,6 +54,7 @@
 
             <intent-filter>
                 <action android:name="android.intent.action.SEND" />
+                <action android:name="app_shortcut" />
```

Comment:
Please use vector drawables; you can import a svg via android studio very easy with option below the normal import option